src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
+const basePrice = 40;
+const threeDays = 3;
+const threeDaysDiscount = 20;
+const sevenDays = 7;
+const sevenDaysDiscount = 50;
```

Comment:
The constant names `threeDays`, `threeDaysDiscount`, `sevenDays`, and `sevenDaysDiscount` could be more descriptive. Consider renaming them to reflect their purpose more clearly, such as `MIN_DAYS_FOR_SHORT_TERM_DISCOUNT`, `SHORT_TERM_DISCOUNT`, `MIN_DAYS_FOR_LONG_TERM_DISCOUNT`, and `LONG_TERM_DISCOUNT` respectively.